core/test/Test/Pos/Core/Gen.hs

```diff
@@ -1057,3 +1062,7 @@ genWord32 = Gen.word32 Range.constantBounded
 
 genWord8 :: Gen Word8
 genWord8 = Gen.word8 Range.constantBounded
+
+
+fixedNM :: NetworkMagic
+fixedNM = NMNothing
```

Comment:
Yeah this should definitely be relocated with the type definition as an alias.

Reply:
https://github.com/input-output-hk/cardano-sl/pull/3558#discussion_r216469998